webapp-client/src/component/CategoryList.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+// TODO: Add support to highlight active category.
```

Comment:
I don't think this needs a TODO. If you really want to, a Github issue would be a better fit for something like this. Also the is-active class would prob be better to add after you implement this highlight active category feature.

Reply:
Added an issue referencing this comment (#19)

Reply:
Please remove this line now that you have an issue for it